### PR TITLE
docs: Improved LightGBM docs

### PR DESCRIPTION
**Pull request by svotaw**
## What changes are proposed in this pull request?

More thorough LightGBM docs

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Word and<br>Phrases Score | Correctness<br>Score | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [website/docs/features/lightgbm/about.md](https://github.com/microsoft/SynapseML/blob/39104fabd5e6ee04a1ebd79cbd6c5fe9db88c3ac/website%2Fdocs%2Ffeatures%2Flightgbm%2Fabout.md) | 82 | 100 | 82 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/6cc0ed3d-6bc5-420b-a897-7984886a59e4) | [:white_check_mark:](# "Successfully processed") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Word and<br>Phrases Score | Correctness<br>Score | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [website/docs/features/lightgbm/about.md](https://github.com/microsoft/SynapseML/blob/a560cb9fed633ebb7c5c4b075a78c5842c5cee8f/website%2Fdocs%2Ffeatures%2Flightgbm%2Fabout.md) | 92 | 100 | 92 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/5bcb490f-d9eb-42c4-b1bd-965071fd6304) | [:white_check_mark:](# "Successfully processed") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Word and<br>Phrases Score | Correctness<br>Score | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [website/docs/features/lightgbm/about.md](https://github.com/microsoft/SynapseML/blob/51c92214f77b82c1adcacc34a4cb51af07e8ac2a/website%2Fdocs%2Ffeatures%2Flightgbm%2Fabout.md) | 95 | 100 | 92 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/fca1fdeb-f66d-4022-b783-79c8a07c97c6) | [:white_check_mark:](# "Successfully processed") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

**Comment by vishalovercome** [comment]
LightGBM not working properly under an auto-scaled environment itself is reason enough to implement GPU support as it will speed up the training time, thereby reducing the chances of task failures

File: website/docs/features/lightgbm/about.md [hunk 1]
```diff
@@ -126,6 +126,42 @@ using `saveNativeModel()`. Additionally, they're fully compatible with [PMML](ht
 can be converted to PMML format through the
 [JPMML-SparkML-LightGBM](https://github.com/alipay/jpmml-sparkml-lightgbm) plugin.
 
+#### Dynamic Scaling Issues
```
Comment (mhamilton723) [inline website/docs/features/lightgbm/about.md:129]: nit: 
```suggestion
#### Dynamic Allocation Limitations
```

Comment (mhamilton723) [reply website/docs/features/lightgbm/about.md:129]: please use the spark teminology: "dynamic allocation" here

File: website/docs/features/lightgbm/about.md [hunk 2]
```diff
@@ -126,6 +126,42 @@ using `saveNativeModel()`. Additionally, they're fully compatible with [PMML](ht
...
+Note: setting any custom configuration can affect cluster startup time if your platform takes advantage of "live pools"
+to improve notebook performance.
+
+If you still have problems, you can consider splitting your data into smaller segments using *numBatches*. Splitting into multiple
+batches increases processing time, but can potentially be used to increase reliability.
+
+#### GPU
+Although LightGBM library can be compiled to support GPUs (through CUDA), SynapseML doesn't currently use a GPU-enabled build. If
+you wish to use GPUs, file an issue with SynapseML so we can measure the interest.
+
```
Comment (mhamilton723) [inline website/docs/features/lightgbm/about.md:164]: nit: maybe lets remove this for now until i talk to ilya

**Review by mhamilton723 — COMMENTED** [review]
Lovely work, only some minor nits

**Comment by svotaw** [comment]
> LightGBM not working properly under an auto-scaled environment itself is reason enough to implement GPU support as it will speed up the training time, thereby reducing the chances of task failures

Ilya will look into using GPU build, but it's not as simple as having a build with CUDA enabled.

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Word and<br>Phrases Score | Correctness<br>Score | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [website/docs/features/lightgbm/about.md](https://github.com/microsoft/SynapseML/blob/d00e47c0ddee053bcbfcfe0d130cc7e5aa67dabb/website%2Fdocs%2Ffeatures%2Flightgbm%2Fabout.md) | 95 | 100 | 92 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/68b1ce19-483b-455c-be13-46b69a608114) | [:white_check_mark:](# "Successfully processed") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Word and<br>Phrases Score | Correctness<br>Score | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [website/docs/features/lightgbm/about.md](https://github.com/microsoft/SynapseML/blob/51e51b981e9ae0bfce8c3398db98bd29f03265db/website%2Fdocs%2Ffeatures%2Flightgbm%2Fabout.md) | 95 | 100 | 92 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/1419c2a1-4eb4-4f47-a4f6-18845d415560) | [:white_check_mark:](# "Successfully processed") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Word and<br>Phrases Score | Correctness<br>Score | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [website/docs/features/lightgbm/about.md](https://github.com/microsoft/SynapseML/blob/ad8fefaf45bf9979ee9ada4d214974b59f398689/website%2Fdocs%2Ffeatures%2Flightgbm%2Fabout.md) | 95 | 100 | 92 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/62e8ccfd-d95c-4120-882c-28119a1ed970) | [:white_check_mark:](# "Successfully processed") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Word and<br>Phrases Score | Correctness<br>Score | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [website/docs/features/lightgbm/about.md](https://github.com/microsoft/SynapseML/blob/2a6b4c6e51d075bd42127928634be48c7893750a/website%2Fdocs%2Ffeatures%2Flightgbm%2Fabout.md) | 95 | 100 | 92 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/306d80e2-fa09-4690-9edb-8ca78823a51e) | [:white_check_mark:](# "Successfully processed") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

**Pull request merged**